From cf93beb47f9907946dd72ca4d481ffbbbadda53a Mon Sep 17 00:00:00 2001
From: Egbert Eich <eich@freedesktop.org>
Date: Mon, 30 May 2011 08:51:58 +0000
Subject: Let <KPDL> key on it,fr and nl keyboards generate KP_Separator

Notwithstanding the never ending discussion what character this
key on the numeric keypad should produce this key should produce
what is imprinted on the keyboards. After all all three countries
are observing the SI standards.
This makes this layout more consistent and match for instance
what is done in the de layout.

Signed-off-by: Egbert Eich <eich@freedesktop.org>
---
diff --git a/symbols/fr b/symbols/fr
index 3898b1c..9c89db8 100644
--- a/symbols/fr
+++ b/symbols/fr
@@ -36,6 +36,8 @@ xkb_symbols "basic" {
     key <AB09>	{ [     colon,      slash, periodcentered,   division ]	};
     key <AB10>	{ [    exclam,    section, dead_belowdot, dead_abovedot ] };
 
+    include "kpdl(comma)"
+
     include "level3(ralt_switch)"
 };
 
diff --git a/symbols/it b/symbols/it
index 3e402c9..9e6ed01 100644
--- a/symbols/it
+++ b/symbols/it
@@ -33,6 +33,8 @@ xkb_symbols "basic" {
 
     key <LSGT>	{ [      less,    greater,guillemotleft,guillemotright] };
 
+    include "kpdl(comma)"
+
     include "level3(ralt_switch)"
 };
 
diff --git a/symbols/nl b/symbols/nl
index ff60dbb..d37ff27 100644
--- a/symbols/nl
+++ b/symbols/nl
@@ -49,6 +49,8 @@ xkb_symbols "basic" {
 
     key <LSGT>	{ [bracketright, bracketleft,       bar,    brokenbar ]	};
 
+    include "kpdl(comma)"
+
     include "level3(ralt_switch)"
 };
 
--
cgit v0.9.0.2-2-gbebe
